src/scripting/flash/system/flashsystem.cpp

```diff
@@ -176,6 +197,117 @@ ASFUNCTIONBODY_ATOM(Capabilities,_getServerString)
 	*/
 	ret = asAtomHandler::fromString(sys,res);
 }
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasAudio)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasAudio always returns true");
+    ret = asAtomHandler::fromBool(true);
+}
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasAudioEncoder)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasAudioEncoder always returns true");
+    ret = asAtomHandler::fromBool(true);
+}
```

Comment:
I think we should return false here as we don't have any support for AudioEncoding implemented.
The same goes for most of the following properties (except hasMP3 and hasTLS)